src/utils/internal.ts

```diff
@@ -1421,7 +1421,17 @@ export async function assertExpectedSqVersion(context: Context): Promise<void> {
     },
   } = await context.queryMiddleware<Ensured<Query, 'subqueryVersions'>>(latestSqVersionQuery());
 
-  if (!sqVersion || lt(sqVersion.version, MINIMUM_SQ_VERSION)) {
+  return sqVersion?.version || '1.0.0';
```

Comment:
Maybe we should cache this value with a TTL? 

From the portal it takes me ~500ms to query middleware.  SQ could get updated so rechecking sometimes makes sense. 

Two round trips for a total of 1 second latency is slow but still usable. On the otherhand this is temporary, so might not be worth the complexity.

Reply:
I guess its only for the few breaking queries though. I was originally thinking it would be before every query.

OK to leave it, but I figure I would mention it.